### ISSUE TITLE
play need histogram for all games played in session

### ISSUE THREAD

**Issue by Wheeler1711** [chw3k5/Wordler#4]
Right now it tells you how many games you have played in the session. I think we want to see a histogram of how well we did for all of the games.

**Comment by chw3k5** [comment]
That makes sense. I should make some sort of user data file to store stats. The hard part will be deciding how we want to display stats without matplotlib.

**Comment by chw3k5** [comment]
The histogram is working and looking good.

**Comment by Wheeler1711** [comment]
Wasn't there an issue in PowerShell on windows?

**Comment by chw3k5** [comment]
That is true. The partial boxes need to be removed and just have integer sized boxes for CMD and PowerShell to look right.
![image](https://user-images.githubusercontent.com/8564941/157099209-5474afb2-e0b4-46ba-9bd9-603a97c59168.png)

**Comment by Wheeler1711** [comment]
I can fix this. I am tempted to leave a non-standard option for the partial boxes and set the default to full boxes.

**Comment by chw3k5** [comment]
You can make it os dependent. like what I do for the `clear_console()` function in https://github.com/chw3k5/Wordler/blob/main/wordler/narrow.py

```
def clear_console():
    command = 'clear'
    if os.name in ('nt', 'dos'):  # If Machine is running on Windows, use cls
        command = 'cls'
    os.system(command)
```

all three of my Windows shells are `os.name=='nt'`
![image](https://user-images.githubusercontent.com/8564941/157123363-1a02aef3-b7a8-4493-9e90-983f2ef3e8e4.png)
